### PR TITLE
Remove preserve egg dir, SHA jinja, bump to 0.4.0

### PR DESCRIPTION
**Pull request by bobleesj**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bobleesj** [comment]
@conda-forge-admin, please rerender

File: README.md
Comment (bobleesj) [inline README.md:1]: Automatically generated

**Comment by bobleesj** [comment]
@sbillinge Could you please merge? Not sure if I have the merge authority as of now.

**Comment by sbillinge** [comment]
> @sbillinge Could you please merge? Not sure if I have the merge authority as of now.

yes, I can merge.  Actually, even if you have merge privileges it is better if you don't merge as this makes a public release, so I would rather make sure for our group software.

**Comment by bobleesj** [comment]
> yes, I can merge. Actually, even if you have merge privileges it is better if you don't merge as this makes a public release, so I would rather make sure for our group software.

Yes, I agree.

**Pull request merged**